library/psa_crypto_core.h

```diff
@@ -208,4 +214,97 @@ psa_status_t psa_copy_key_material_into_slot( psa_key_slot_t *slot,
  */
 psa_status_t mbedtls_to_psa_error( int ret );
 
+/** Get the PSA CTR_DRBG context
+ *
+ * \return A pointer to the PSA CTR_DRBG context
+ */
+mbedtls_ctr_drbg_context *psa_get_ctr_drbg_context( void );
+
+/** Import a key in binary format.
+ *
+ * \note The signature of this function is that of a PSA driver
+ *       import_key entry point. This function behaves as an import_key
+ *       entry point as defined in the PSA driver interface specification for
+ *       transparent drivers.
+ *
+ * \param[in]  attributes       The attributes for the key to import.
+ * \param[in]  data             The buffer containing the key data in import
+ *                              format.
+ * \param[in]  data_length      Size of the \p data buffer in bytes.
+ * \param[out] key_buffer       The buffer containing the key data in output
```

Comment:
Minor: would be clearer stating "The buffer to contain the key data in output format upon successful return" since the buffer isn't supposed to contain anything on function entry.

I know it's implied by marking it as an output, but still.